plugins/workbench/src/components/internal/CreateForm.svelte

```diff
@@ -1,118 +1,116 @@
 <!--
-// Copyright © 2020 Anticrm Platform Contributors.
-//
-// Licensed under the Eclipse Public License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License. You may
-// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//
-// See the License for the specific language governing permissions and
-// limitations under the License.
+Copyright © 2020, 2021 Anticrm Platform Contributors.
+
+Licensed under the Eclipse Public License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may
+obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+See the License for the specific language governing permissions and
+limitations under the License.
 -->
 <script lang="ts">
-  import { Class, Doc, Ref } from '@anticrm/core'
   import { createEventDispatcher } from 'svelte'
-  import { AnyComponent } from '@anticrm/platform-ui'
+
+  import { Ref } from '@anticrm/core'
+  import { CORE_CLASS_VDOC, Space } from '@anticrm/domains'
   import presentation, {
-    AttrModel, ClassModel, getComponentExtension, getPresentationService, getCoreService
+    AttrModel,
+    ClassModel,
+    getComponentExtension,
+    getPresentationService
   } from '@anticrm/presentation'
-  import Properties from '@anticrm/presentation/src/components/internal/Properties.svelte'
+  import { AnyComponent } from '@anticrm/platform-ui'
+  import workbench, { ItemCreator } from '@anticrm/workbench'
+
   import Icon from '@anticrm/platform-ui/src/components/Icon.svelte'
-  import workbench from '@anticrm/workbench'
-  import InlineEdit from '@anticrm/sparkling-controls/src/InlineEdit.svelte'
-  import Button from '@anticrm/sparkling-controls/src/Button.svelte'
-  import { CORE_CLASS_VDOC, Space } from '@anticrm/domains'
   import Component from '@anticrm/platform-ui/src/components/Component.svelte'
 
-  export let title: string = ''
-  export let _class: Ref<Class<Doc>>
-  export let space: Ref<Space>
-  let object = {} as any
+  import DefaultForm from './DefaultForm.svelte'
 
-  const coreService = getCoreService()
+  export let creator: ItemCreator
+  export let space: Ref<Space> | undefined
```

Comment:
I'm not very happy we have this pair passed in multiple places.

Could we pass only (Space | undefined) and have a list of Spaces to be inside svelte context from top level Application component? Or retrieved in case they are needed.